Mathlib/Algebra/CubicDiscriminant.lean

```diff
@@ -40,8 +40,6 @@ structure Cubic (R : Type*) where
 
 namespace Cubic
 
-open Cubic Polynomial
```

Comment:
I wonder if one of these was `open scoped` (during the port, for some time `open`s were introduced that should have been `open scoped`). Can you check, please?